packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -280,34 +301,58 @@ function checkForGraphFragmentPattern(
     // should be differences other than omitting non-reference properties.
 
     if (!fromApplicator && !variantOmitsProperties) {
-      const variantIncludesAll = schemaHasConstraint(canonical, c => {
+      logger.debug(
+        `${ruleId}: checking that variant schema is a proper subset of canonical schema`
+      );
+
+      const propertyOmitted = schemaHasConstraint(canonical, c => {
         if (c && isObject(c.properties)) {
-          let allAreIncluded = true;
+          let omitted = false;
           for (const [name, prop] of Object.entries(c.properties)) {
-            const included = schemaHasConstraint(
-              variant,
-              s =>
+            logger.debug(`${ruleId}: checking canonical property '${name}'`);
+
+            const included = schemaHasLooseConstraint(variant, s => {
+              return (
```

Comment:
This `return` is probably unnecessary - you could get rid of the brackets and use pure arrow notation, which returns implicitly.